xendit-android/src/main/java/com/xendit/Xendit.java

```diff
@@ -265,10 +265,10 @@ public String getCardKey() {
      * @param tokenCallback The callback that will be called when the token creation completes or
      *                      fails
      */
-    public void createSingleUseToken(final Card card, final int amount, final TokenCallback tokenCallback) {
+    public void createSingleUseToken(final Card card, final int amount, final boolean shouldAuthenticate, final TokenCallback tokenCallback) {
```

Comment:
Actually, leave this method signature as is, default can be true or undefined. And create a new signature that takes should authenticate